cmd/watch-termination/main.go

```diff
@@ -131,8 +131,11 @@ func run() int {
 
 		if *gracefulTerminatioPeriod > 2*time.Second {
 			go func() {
-				<-termCh
-				<-time.After(*gracefulTerminatioPeriod - 2*time.Second)
+				select {
+				case <-termCh:
```

Comment:
`termCh` is closed when we receive a signal (for example `SIGINT`) and then graceful termination starts. We expect the process to exit within `gracefulTerminatioPeriod`.

The previous code, waited until the signal was received and then `gracefulTerminatioPeriod - 2*time.Second` to signal that the process hasn't terminated and will be forcefully killed.

```
<-termCh
<-time.After(*gracefulTerminatioPeriod - 2*time.Second)
```

The new code will yield after `gracefulTerminatioPeriod - 2*time.Second` (unless `termCh` is closed before) and 
will signal that the process hasn't been terminated which is not what we want.

```
select {
 case <-termCh:
   return
  case <-time.After(*gracefulTerminatioPeriod - 2*time.Second):
}
```

Reply:
`s/termCh/gracefulTermStartedCh/` would avoid the confusion.